### PR TITLE
Feature/move visgeo file

### PR DESCRIPTION
**Pull request by meganrm**
Problem
=======
I had held off moving VisGeometry so that the diff wasn't too complicated when it had a lot of edits

Solution
========
Moved and renamed VisGeometry

## Type of change
Please delete options that are not relevant.

* Maintenance

Steps to Verify:
----------------
1. npm start
2. test app should work normally

### CODE REVIEW

File: src/simularium/VisGeometry/index.ts [hunk 1]
```diff
@@ -29,32 +29,35 @@ import jsLogger from "js-logger";
...
-import { InstancedMesh } from "./rendering/InstancedMesh";
-import { LegacyRenderer } from "./rendering/LegacyRenderer";
-import GeometryStore, { DEFAULT_MESH_NAME } from "./VisGeometry/GeometryStore";
+} from "../types";
+import { AgentData } from "../VisData";
+
+import SimulariumRenderer from "../rendering/SimulariumRenderer";
+import { InstancedFiberGroup } from "../rendering/InstancedFiber";
+import { InstancedMesh } from "../rendering/InstancedMesh";
+import { LegacyRenderer } from "../rendering/LegacyRenderer";
```
Comment (toloudis) [inline src/simularium/VisGeometry/index.ts:52]: VisGeometry is the only one that imports all these `rendering/` files. It looks odd that they became further removed from VisGeometry.  Do you have any objection to my doing a followup that brings all the visualization code together and starts to separate it from the controller/simulation code?  Or doing it in this PR.

Comment (meganrm) [reply src/simularium/VisGeometry/index.ts:52]: you mean just moving the rendering folder into VisGeometry/ too? Or more extensive changes. The former seems fine for this PR, the latter maybe separate PR

Comment (toloudis) [reply src/simularium/VisGeometry/index.ts:52]: Yes - please just move rendering/ into VisGeometry and we can do the rest in a separate PR! thank you

Comment (meganrm) [reply src/simularium/VisGeometry/index.ts:52]: The `worker/KMeansWorker` imports `rendering/KMeans3d` just FYI

File: package.json [hunk 2]
```diff
@@ -49,6 +49,7 @@
         "@types/react": "^16.14.3",
         "@typescript-eslint/eslint-plugin": "~4.29.0",
         "@typescript-eslint/parser": "~4.29.0",
+        "@webpack-cli/serve": "^1.5.2",
```
Comment (toloudis) [inline package.json:52]: can remove?

**Review by toloudis — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (meganrm) [reply package.json:52]: I needed it to run `npm start`, is it just assumed people already have this installed?

Comment (toloudis) [reply package.json:52]: I have not needed it yet, so not sure.  I'm surprised there wouldn't also be a package-lock update then too.

**Review by schoinh — APPROVED** [review]

**Pull request merged**